refinery/core/models.py

```diff
@@ -899,15 +899,15 @@ def get_node_collection(self):
                 NodeCollection.MultipleObjectsReturned) as exc:
             logger.error('Failed to get NodeCollection for '
                          'Investigation %s: %s',
-                         unicode(self.investigation), exc)
+                         str(self.investigation), exc)
```

Comment:
Explicit cast to string is probably unnecessary. Also, this whole statement can be compacted to two lines with sacrificing readability.